src/react/actions/sts.js

```diff
@@ -21,6 +21,7 @@ export function assumeRoleWithWebIdentity(roleArn: string): ThunkStatePromisedAc
                     sessionToken: creds.Credentials.SessionToken,
                 };
                 zenkoClient.login(params);
+                iamClient.init(params);
```

Comment:
We might want to call the method `login` to be consistant.

Reply:
done